neuron_morphology/snap_polygons/image_outputter.py

```diff
@@ -24,7 +24,7 @@ def __init__(
         self, 
         native_geo: Geometries, 
         result_geo: Geometries, 
-        image_specs: Sequence[Dict],
+        image_specs: Optional[Sequence[Dict]],
```

Comment:
This can be optional, which is good, but the function in `__main__.run_snap_polygons()` will still have an `images` key in its output dictionary. Not sure it's better to have that (with the value of an empty list, as happens with this PR) or not to include that key-value pair at all?

Reply:
I think it is (very slightly) better to have the key. The only meaningful change seems to be that parsing code can be a bit cleaner, since it doesn't need to branch on the presence of the `"images"` key. Asking whether images are present is still very quick, since you just as whether the list is empty.

Reply:
Sounds good to me.